Core/Frameworks/Baikal/Resources/Templates/Navigation/Topbar/Anonymous.html

```diff
@@ -0,0 +1,20 @@
+{% autoescape false %}
+<div class="navbar navbar-inverse navbar-fixed-top">
+	<div class="navbar-inner">
+		<div class="container">
+			<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</a>
+			<a class="brand" href="{{ homelink }}"><img style="vertical-align: text-top; line-height: 20px;" src="res/core/BaikalAdmin/Templates/Page/baikal-text-20.png" /> Server</a>
+			<div class="nav-collapse">
+				<ul class="nav">
+					<li> <a href="admin/"><i class="icon-eject icon-white" style="transform: rotateX(150deg)"></i> Login</a></li>
```

Comment:
What do you think about using `transform: rotate(90deg);` instead? In my opinion, this looks more like a login icon.

![image](https://user-images.githubusercontent.com/5811634/56531058-acf5ed80-6553-11e9-8357-97c3207ea83a.png)

Reply:
changed in 0934fab4f0fdb70fe541f47655ffd6b6d45036cc
Yep, way better =)